samples/sample-junit5/build.gradle.kts

```diff
@@ -1,7 +1,8 @@
 plugins {
     java
     jacoco
-    id("org.gradle.test-retry") version "1.0.0"
+    //TODO this needs to replaced as soon as 1.1.7 is out together with junit5.sample.out#L9 and remove `--warning-mode all` from junit5.sample.confL2
+    id("org.gradle.test-retry") version "1.1.6"
```

Comment:
❓ Why do we need to declare a version here? Shouldn't we inject to the `GradleRunner` via the classpath?

Reply:
I am not sure how to apply the `GradleRunner` in that context. Its a sample project executed via [Examplar](https://github.com/gradle/exemplar) - Nevertheless the latest test-retry-plugin is now always applied.